src/Fabulous.XamarinForms/Views/Cells/ImageCell.fs

```diff
@@ -0,0 +1,55 @@
+namespace Fabulous.XamarinForms
+
+open System
+open System.IO
+open Fabulous
+open Fabulous.XamarinForms
+open Xamarin.Forms
+
+type IImageCell =
+    inherit ITextCell
+
+module ImageCell =
+    let WidgetKey = Widgets.register<ImageCell> ()
+
+    let ImageSource =
+        Attributes.defineAppThemeBindable<ImageSource> ImageCell.ImageSourceProperty
+
+[<AutoOpen>]
+module ImageCellBuilders =
+    type Fabulous.XamarinForms.View with
+        static member inline ImageCell<'msg>(light: ImageSource, ?dark: ImageSource) =
```

Comment:
Here as well. I would put both the light/dark and the text value as mandatory

Reply:
Done 😀